demisto_sdk/commands/lint/helpers.py

```diff
@@ -62,6 +62,8 @@
 # Line break
 RL = "\n"
 
+IMPORT_API_MODULE_REGEX = r"from ([\w\d]+ApiModule) import \*(?:  # noqa: E402)?"
```

Comment:
```suggestion
IMPORT_API_MODULE_REGEX = r"from (\w+ApiModule) import \*(?:  # noqa: E402)?"
```
`\d` is contained in `\w`